backend/src/test/java/com/woowacourse/momo/acceptance/participant/ParticipantAcceptanceTest.java

```diff
@@ -62,10 +63,20 @@ void participateByNonMember() {
         모임에_참여한다(groupId).statusCode(HttpStatus.BAD_REQUEST.value()); // TODO: UNAUTHORIZED
     }
 
+    @DisplayName("정원이 가득 찬 모임에 참여한다")
+    @Test
+    void participateFullGroup() {
+        String accessToken = MOMO.로_로그인한다();
+        모임에_참여한다(accessToken, groupId);
+
+        accessToken = GUGU.로_로그인한다();
+        모임에_참여한다(accessToken, groupId).statusCode(HttpStatus.BAD_REQUEST.value());
+    }
```

Comment:
```suggestion
    void participateFullGroup() {
        String accessToken = MOMO.로_로그인한다();
        모임에_참여한다(accessToken, groupId).statusCode(HttpStatus.OK.value());

        accessToken = GUGU.로_로그인한다();
        모임에_참여한다(accessToken, groupId).statusCode(HttpStatus.BAD_REQUEST.value());
    }
```
첫번째 요청은 정상적으로 OK가 됐는지 확인하면, 시나리오가 보다 명확해보여요!!

Reply:
해당 내용도 반영했습니다!!